exercise-book/src/nrf52-troubleshoot-cargo-run-error.md

```diff
@@ -8,7 +8,7 @@ You may get one of these errors:
 ```console
 $ cargo run --bin usb-4
 Running `probe-rs run --chip nRF52840_xxAA target/thumbv7em-none-eabihf/debug/usb-4`
-Error: An error specific to a probe type occured: USB error while taking control over USB device: Access denied (insufficient permissions)
```

Comment:
Please don't fix typos in console output - the program output is what it is

Reply:
Ack.